contracts/Voting.sol

```diff
@@ -440,34 +453,39 @@ contract Voting is IForwarder, AragonApp, BasicMetaTransaction {
         lastCreateVoteTimes[msgSender()] = getTimestamp64();
 
         if (_castVote && _canVote(voteId, msgSender())) {
-            _vote(voteId, true, msgSender(), _executesIfDecided);
+            _vote(voteId, PCT_BASE, msgSender(), _executesIfDecided);
         }
     }
 
     /**
     * @dev Internal function to cast a vote. It assumes the queried vote exists.
     */
-    function _vote(uint256 _voteId, bool _supports, address _voter, bool _executesIfDecided) internal {
+    function _vote(uint256 _voteId, uint256 _pct, address _voter, bool _executesIfDecided) internal {
         Vote storage vote_ = votes[_voteId];
 
         VoterState state = vote_.voters[_voter];
         require(state == VoterState.Absent, "Can't change votes");
+        require(_pct != PCT_BASE / 2, "VOTING_CANNOT_ABSTAIN");
```

Comment:
What do you think of following the convention of having the error string as a `private constant`?

Reply:
Maybe we can also include a comment with the explanation of why this require prevents voting 50%.

Reply:
I'm worried to not add extra variables, but I understand constants do not count as them right?